Src/Support/Google.Apis.Auth.Tests/OAuth2/LimitedLocalhostHttpServerTests.cs

```diff
@@ -29,10 +29,13 @@ namespace Google.Apis.Auth.Tests.OAuth2
 {
     public class LimitedLocalhostHttpServerTests
     {
-        private LocalServerCodeReceiver.LimitedLocalhostHttpServer StartServer()
+        private const string CustomResponseHtml = "<html><body>Custom response!</body</html>";
```

Comment:
Missing `>` from `</body>`. I know the HTML itself is irrelevant, but....

Reply:
done